src/applications/vaos/tests/appointment-list/components/PastAppointmentsListV2.unit.spec.jsx

```diff
@@ -287,6 +289,51 @@ describe('VAOS <PastAppointmentsListV2>', () => {
     expect(within(firstCard).getByText(/VA Video Connect at home/i)).to.exist;
   });
 
+  it('should display past appointments using V2 api call', async () => {
+    const now = moment().startOf('day');
+    const start = moment(now).subtract(3, 'months');
+    const end = moment()
+      .minutes(0)
+      .add(30, 'minutes');
+
+    const yesterday = moment().subtract(1, 'day');
+    const appointment = getVAOSRequestMock();
+    appointment.id = '123';
+    appointment.attributes = {
+      ...appointment.attributes,
+      kind: 'phone',
+      minutesDuration: 30,
+      status: 'booked',
+      start: yesterday.format('YYYY-MM-DDTHH:mm:ss'),
+    };
+    mockVAOSAppointmentsFetch({
+      start: start.format('YYYY-MM-DDTHH:mm:ssZ'),
+      end: end.format('YYYY-MM-DDTHH:mm:ssZ'),
+      requests: [appointment],
+      statuses: ['booked'],
+    });
+
+    const myInitialState = {
+      ...initialState,
+      featureToggles: {
+        ...initialState.featureToggles,
+        vaOnlineSchedulingVAOSServiceVAAppointments: true,
+      },
+    };
+    const screen = renderWithStoreAndRouter(<PastAppointmentsListV2 />, {
+      initialState: myInitialState,
+    });
+
+    await screen.findAllByText(
+      new RegExp(yesterday.format('dddd, MMMM D'), 'i'),
+    );
+
+    expect(screen.queryByText(/You don’t have any appointments/i)).not.to.exist;
+    expect(global.fetch.firstCall.args[0]).to.equal(
```

Comment:
I'm not sure this check is necessary. If we mess up the fetch call, there won't be any appointments and the expect statement one line above this will fail. I think this would apply to the other tests that are checking the fetch call. If they break, other assertions in the test will fail, because things won't be mocked properly.